packages/blocks/.eslintrc.cjs

```diff
@@ -12,6 +12,15 @@ module.exports = {
     node: true,
   },
   rules: {
+    // @todo(mp APP-2482): move to js-config
     'sonarjs/no-duplicate-string': 'off',
+    'no-undef-init': 'off',
+    'unicorn/no-useless-undefined': 'off',
+
+    // @todo(mp APP-2482): diagnose why these aren't working correctly
+    '@typescript-eslint/no-unsafe-assignment': 'off',
+    '@typescript-eslint/no-unsafe-return': 'off',
+    '@typescript-eslint/no-unsafe-member-access': 'off',
+    '@typescript-eslint/no-unsafe-call': 'off',
```

Comment:
Our TS eslint plugin is not able to type check svelte components. Added a ticket to investigate [here](https://viam.atlassian.net/browse/APP-2482) and turned off for now since svelte-check will still catch these since we're in TS strict mode.

Reply:
> out TS eslint plugin is not able to type check svelte components

Weird that it only seems to be happening with the `on:create` arguments of various `T.[]` components. If this was a general problem I'd expect to see a bunch more errors, both here and in app, which uses the same ESLint config.

Perhaps it's safer to use ignore comments rather than disable the rules wholesale, since it seems to be limited to `@threlte/core`?

Reply:
Found a workaround! I think the issue is related to how there are currently type-checking issues in Svelte's template sections. I was able to restore all rules by just passing references to functions that were typed in the script section, and that cleared everything up.